.github/workflows/reusable-renovate-workflow.yml

```diff
@@ -50,6 +50,16 @@ jobs:
           keyvault-name: ${{ vars.WORKLEAP_GLOBAL_KEYVAULT_NAME }}
           secret-name: "renovate-tfc-token"
 
+      - name: Get renovate-tfc-token secret
+        id: get_ado_feeds_package_read_secret
+        uses: workleap/wl-reusable-workflows/retrieve-managed-secret@main
+        with:
+          azure-client-id: ${{ vars.AZURE_CLIENT_ID }}
+          azure-tenant-id: ${{ vars.AZURE_TENANT_ID }}
+          azure-subscription-id: ${{ vars.AZURE_SUBSCRIPTION_ID }}
+          keyvault-name: ${{ vars.IDP_CICD_KEYVAULT_NAME }}
```

Comment:
Not sure team will have access to thiis KV. We should put the secret in: WORKLEAP_GLOBAL_KEYVAULT_NAME

Reply:
With the subscriptions WORKLEAP_GLOBAL_KEYVAULT_SUBSCRIPTION_ID

<img width="860" height="473" alt="Image" src="https://github.com/user-attachments/assets/823abd7d-c53a-48bf-9d44-b3cfacd126f5" />